app/app.go

```diff
@@ -43,7 +43,7 @@ func (app *fyneApp) UniqueID() string {
 	}
 
 	fyne.LogError("Preferences API requires a unique ID, use app.NewWithID()", nil)
-	app.uniqueID = fmt.Sprintf("missing-id-%d", time.Now().Unix()) // This is a fake unique - it just has to not be reused...
+	app.uniqueID = "missing-id-" + strconv.FormatInt(time.Now().Unix(), 10) // This is a fake unique - it just has to not be reused...
```

Comment:
Is there a reason you didn’t use `ItoA` here?

Reply:
Because `time.Now().Unix()` is of type `int64` and that is what `strconv.FormatInt()` takes. Using `strconv.Itoa()` would mean that we would have converted it from `int64` to `int` and then back into `int64` (Itoa is a wrapper around FormatInt).